src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/TestHelper.java

```diff
@@ -113,7 +115,7 @@ public String getSasToken(String containerName, String testUrl) throws Exception
             .relaxedHTTPSValidation()
             .baseUri(testUrl)
             .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
-            .header(SyntheticHeaders.SYNTHETIC_TEST_SOURCE, "Bulk Scan Processor functional test")
+            .header(SyntheticHeaders.SYNTHETIC_TEST_SOURCE, TEST_SOURCE_NAME)
```

Comment:
can we add method name or sth defines the test so when it fails we can track it easily
like TEST_SOURCE_NAME + " s2sSignIn"

Reply:
We'll find the value from this header in AppInsights requests, along with information about which endpoints were hit.